src/models/User/createUser/index.js

```diff
@@ -43,8 +43,8 @@ const createUser = async ({
   const existingUsersWithEmail = await User.query()
     .where({ email })
     .count()
-
-  const isEmailInUse = existingUsersWithEmail[0].count > 0
+    .first()
```

Comment:
objection doesn’t improve `count()`. Let’s use `first()` :)

Reply:
Objection handles all Knex QueryBuilder methods so count should work in this case. Or maybe we could use findOne for those cases where we need to check if an element exists in db already ?

Reply:
Sorry I wasn’t clear. I meant `count().first()` instead of `count()[0]`. Obviously we need to use `count()`.
But it still returns an array with the first element being the count. Which is weird.

```
Or maybe we could use findOne for those cases where we need to check if an element exists in db already ?
```

A count is way more performant on the database level, no?

Reply:
In this specific case we could use a 
```
User.query().findOne({email})
```
in order to get a single object from db (it uses a SQL `LIMIT` clause).

It would make this simpler, you would just need to test if `existingUsersWithEmail` is `undefined` , wdyt ?

Reply:
Yeah, I understood what you meant. I just think that, in general, if we don’t want to actually use the user that is returned but just want to know if it exists, using a `count()` is the correct way to do it.

But yeah, we can go ahead and use this instead in both cases, I’m not too precious about it.

Reply:
You're right count() would make more sense, let's use .first() as you originally did then